octoprint_mrbeam/util/uptime.py

```diff
@@ -1,14 +1,19 @@
 # coding=utf-8
-
+from sys import platform
 from datetime import timedelta
 from octoprint_mrbeam.mrb_logger import mrb_logger
 
 # http://planzero.org/blog/2012/01/26/system_uptime_in_python,_a_better_way
 def get_uptime():
     try:
-        with open("/proc/uptime", "r") as f:
-            uptime = float(f.readline().split()[0])
-        return uptime
+        if platform == 'darwin':
+            raise NotImplementedError('uptime not implemented for darwin')
```

Comment:
on Mac, you can just call `uptime`. 
The resulting output has this format: 
> 10:33 up 21:40, 4 users, load averages: 0.09 0.19 0.21
( https://osxdaily.com/2009/09/22/check-your-macs-uptime-and-reboot-history/ )

Reply:
can we use this instead of handling each manually? https://pythonhosted.org/uptime/ @Gallore

Reply:
For technical reasons, the uptime package is not installed on our devices and will not be installed yet. You can add the `uptime` package in `setup.py` as an optional requirements for `[develop]`

Reply:
I'm fine with either way.